trino/client.py

```diff
@@ -66,6 +66,7 @@
     PROXIES = {}
 
 _HEADER_EXTRA_CREDENTIAL_KEY_REGEX = re.compile(r'^\S[^\s=]*$')
+PYTHON_MAX_DECIMAL_PRECISION = 6
```

Comment:
Rename to `MAX_PYTHON_TEMPORAL_PRECISION_POWER`, move to line 77 and reuse in constant `MAX_PYTHON_TEMPORAL_PRECISION`.

Reply:
Done